docs/content/preview/api/ycql/function_datetime.md

```diff
@@ -117,6 +117,98 @@ ycqlsh:example> SELECT todate(ts) FROM test_todate;
  2018-10-09
 ```
 
+## minTimeUUID(<timestamp>)
+
+This function generates corresponding (`TIMEUUID`) with minimum node/clock component so that it includes all regular 
+`TIMEUUID` with that timestamp when comparing with another `TIMEUUID`.
+
+- It takes in an argument of type `TIMESTAMP`. 
+- The return value is a `TIMEUUID`.
+
+### Examples
+
+#### Insert values using now()
+
+```sql
+ycqlsh:example> CREATE TABLE test_min (k INT PRIMARY KEY, v TIMEUUID);
+```
+
+```sql
+ycqlsh:example> INSERT INTO test_min (k, v) VALUES (1, now());
+```
+
+```sql
+ycqlsh:ybdemo> select k, v, totimestamp(v) from test_min;
+```
+```sql
+ k | v                                    | totimestamp(v)
+---+--------------------------------------+---------------------------------
+ 1 | dc79344c-cb79-11ec-915e-5219fa422f77 | 2022-05-04 07:14:39.205000+0000
+
+(1 rows)
+```
+
+#### Select using minTimeUUID()
+
+```sql
+ycqlsh:ybdemo> SELECT * FROM test_min WHERE v > minTimeUUID('2022-04-04 13:42:00+0000');
+```
+
+```sql
+ k | v
+---+--------------------------------------
+ 1 | dc79344c-cb79-11ec-915e-5219fa422f77
+
+(1 rows)
+```
+
+## maxTimeUUID(<timestamp>)
+
+This function generates corresponding (`TIMEUUID`) with maximum node/clock component so that it includes all regular 
+`TIMEUUID` with that timestamp when comparing with another `TIMEUUID`.
+
+
+- It takes in an argument of type `TIMESTAMP`. 
+- The return value is a `TIMEUUID`.
+
+### Examples
+
+#### Insert values using now()
+
+```sql
+ycqlsh:example> CREATE TABLE test_max (k INT PRIMARY KEY, v TIMEUUID);
+```
+
+```sql
+ycqlsh:example> INSERT INTO test_max (k, v) VALUES (1, now());
+```
+
+```sql
+ycqlsh:ybdemo> select k, v, totimestamp(v) from test_max;
+```
+
+```sql
+ k | v                                    | totimestamp(v)
+---+--------------------------------------+---------------------------------
+ 1 | e9261bcc-395a-11eb-9edc-112a0241eb23 | 2020-12-08 13:40:18.636000+0000
+
+(1 rows)
+```
+
+#### Select using maxTimeUUID()
+
+```sql
+ycqlsh:ybdemo> SELECT * FROM test_min WHERE v <= maxTimeUUID('2022-05-05 00:34:32+0000');
+```
+
+```sql
+ k | v
+---+--------------------------------------
```

Comment:
This code block should be tagged as output (to disable the copy action)

Reply:
fixed all cases